agent/stats/engine.go

```diff
@@ -769,10 +770,11 @@ func (engine *DockerStatsEngine) ContainerDockerStats(taskARN string, containerI
 		if ok {
 			taskNetworkStats := taskStats.StatsQueue.GetLastStat().Networks
 			containerStats.Networks = taskNetworkStats
+			containerNetworkRateStats = taskStats.StatsQueue.GetLastNetworkStatPerSec()
```

Comment:
shouldn't this reuse the variable containerNetworkRateStats?

Reply:
It will get overwritten.

Reply:
oh i totally missed that.